lib/utils.js

```diff
@@ -17,6 +17,7 @@
  */
 
 const { 'default': Axios } = require('axios');
+const { admin } = require('googleapis/build/src/apis/admin');
```

Comment:
Why is this imported?

Reply:
This was a mistake, I will fix it in the next commit.